Installing on OS X Mavericks

Hi!
I have problem installing pyfftw on OS X. I have installed fftw3 through pkgin (http://saveosx.org).

Some details:
libfftw3.dylib is found in/usr/pkg/lib
fftw3.h is found in /usr/pkg/includes

Just using `pip install pyfftw` yields `pyfftw.c:260:10: fatal error: 'fftw3.h' file not found`. Adding `C_INCLUDE_PATH=/usr/pkg/include/` gets a little longer:

```
cc -bundle -undefined dynamic_lookup -arch x86_64 -arch i386 -Wl,-F. -Qunused-arguments -Qunused-arguments build/temp.macosx-10.9-intel-2.7/Users/arve/Dokumenter/git/FY3490/.vs.env/build/pyfftw/pyfftw/pyfftw.o -lfftw3 -lfftw3f -lfftw3l -lfftw3_threads -lfftw3f_threads -lfftw3l_threads -o build/lib.macosx-10.9-intel-2.7/pyfftw/pyfftw.so

ld: library not found for -lfftw3
```

My LD environment variables:

```
LD_LIBRARY_PATH=/usr/pkg/lib
DYLD_LIBRARY_PATH=/usr/pkg/lib
```

As a side note, PyFFTW3 installs find with `DYLD_LIBRARY_PATH` set.

Any suggestions?

--- a/README.rst
+++ b/README.rst
@@ -114,7 +114,7 @@ Now install pyfftw from pip::
 
   pip install pyfftw
 
-Notes: pkgin fftw package does not contain long or float impletations of fftw, 
+Notes: `pkgin <http://saveosx.org>`_ fftw package does not contain long or float impletations of fftw, 
 and installation will fail.
 
 It has been suggested that `macports <http://www.macports.org/>`_ might also 

